core/field_number.ts

```diff
@@ -291,14 +291,17 @@ export class FieldNumber extends FieldInput<number> {
    *
    * @returns The newly created number input editor.
    */
-  protected override widgetCreate_(): HTMLElement {
-    const htmlInput = super.widgetCreate_();
+  protected override widgetCreate_(): HTMLInputElement {
+    const htmlInput = super.widgetCreate_() as HTMLInputElement;
```

Comment:
Can we have the super class return `HTMLInputElement` since it is the one making the input element?

Reply:
Unfortunately that doesn't work because FieldMultilineInput overrides it to return HTMLTextAreaElement, and that's not a subclass of HTMLInputElement.